main/src/main/scala/sbt/internal/librarymanagement/CoursierInputsTasks.scala

```diff
@@ -58,16 +58,14 @@ private[sbt] object CoursierInputsTasks {
       sv,
       sbv
     )
-    val proj1 = proj0.copy(
-      dependencies = proj0.dependencies.map {
-        case (config, dep) =>
-          (config, dep.copy(exclusions = dep.exclusions ++ exclusions0))
-      }
-    )
+    val proj1 = proj0.withDependencies(proj0.dependencies.map {
```

Comment:
did a case class get converted to contraband class?

Reply:
Yea. For bincompat - https://github.com/coursier/sbt-coursier/pull/73